Index.html

```diff
@@ -1,14 +1,78 @@
 <!doctype html>
 <html>
 <head>
-    <link href="#" rel="stylesheet">
+    
     <link href="assets/css/Styles.css" rel="stylesheet" type="text/css">
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="icon" href="#">
+    <link rel="icon" href="assets/images/githubsvg.svg">
+    <title>CopyCat GitHub</title>
 </head>
 
 <body>
+    <header>
+    <div class="container-nav">
+        <nav>
+            <ul class="float-left">
+             <li><img class="border" src="assets/images/githubsvg.svg" alt="github-logo" height="40px" width="40px"></li> 
+                <li><a href="#">Why GitHub?&nbsp;<span class="rotate-180">^</span></a>
+                    <ul>
+                        <li><a href="#"><b>Features</b></a></li>
```

Comment:
Can we change the `<b>` tag to `<strong>` please, `<b>` has been deprecated and replaced with `<strong>`